api/go.mod

```diff
@@ -10,5 +10,6 @@ require (
 replace (
 	github.com/mattn/go-sqlite3 => github.com/mattn/go-sqlite3 v1.14.18
 	golang.org/x/net => golang.org/x/net v0.17.0
+	golang.org/x/net v0.17.0 => golang.org/x/net v0.23.0
```

Comment:
does this not just reduce the above line down to 
`golang.org/x/net => golang.org/x/net v0.23.0`?